spec/helpers/content_profiles.ts

```diff
@@ -153,6 +158,49 @@ class ContentProfiles {
                 .first()
                 .element(by.tagName('button')).click();
         };
+
+        /**
+         * Open content profile dropdown.
+        */
+        this.openContentProfileDropdown = () => {
+            element(by.id('content-profile-dropdown')).click();
+        };
+
+        function selectProfile(filtered) {
+            if (filtered.length) {
+                return filtered[1].click();
+            }
+        }
+
+        /**
+         * Returns the list of content profiles.
+        */
+        this.totalProfiles = () => {
+            const dropdownMenu = element(by.id('select-content-profile'));
+
+            return dropdownMenu.all(by.repeater('profile in aggregate.activeProfiles'));
```

Comment:
Only query DOM elements using `data-test-id` markers for testing. Don't query by ids, class names, repeaters, etc. See testing section at https://github.com/superdesk/superdesk-client-core/blob/develop/CONTRIBUTING.md

There are also helpers easily get elements by test-id like `el(['monitoring', 'clear-filters]).click()`. You can take a look at `spec/internal_destinations_spec.ts` for examples of how to use it.